app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -161,11 +161,9 @@ def pull_manifest
       end
 
       if (manifest_response = redirect_client { Resources::Registry::Proxy.get(@_request.fullpath, headers) })
-        #when pulp 2 is removed, this should no longer be needed, and all clients should be redirected
-        logger.debug filter_sensitive_data(manifest_response)
+        #for some requests, we get a redirect, but for others we get the actual manifest in response
```

Comment:
also i discovered that pulp doesn't always redirect on manifest requests, so i changed the comment here.